source/extensions/common/opentelemetry/sdk/trace/types.h

```diff
@@ -0,0 +1,42 @@
+#pragma once
+
+#include <memory>
+
+#include "opentelemetry/proto/collector/trace/v1/trace_service.pb.h"
+#include "opentelemetry/proto/trace/v1/trace.pb.h"
+
+namespace Envoy {
+namespace Extensions {
+namespace Common {
+namespace OpenTelemetry {
+namespace Sdk {
+namespace Trace {
+
+/**
+ * OpenTelemetry trace signal type aliases.
+ *
+ * Origin: All types are derived from official OpenTelemetry C++ SDK and protocol definitions
+ * Reference: https://github.com/open-telemetry/opentelemetry-cpp
+ */
+
+/**
+ * @brief The type of the span (from OpenTelemetry specification)
+ * @see
+ * https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#spankind
+ */
+using OTelSpanKind = ::opentelemetry::proto::trace::v1::Span::SpanKind;
+
+// OTLP export request/response types for traces (from OTLP protocol)
+using ExportRequest = ::opentelemetry::proto::collector::trace::v1::ExportTraceServiceRequest;
```

Comment:
My general feeling is that we only gain benefits from a constant is when there is more than one usage. For these RPC stubs, there's really only one place they are used, which is the various sinks.